src/Elders.Cronus/AtomicAction/AggregateStateFirstLevelConcurrencyException.cs

```diff
@@ -3,9 +3,6 @@
 
 namespace Elders.Cronus.AtomicAction
 {
-    /// <summary>
-    /// 
-    /// </summary>
     [Serializable]
     public class AggregateStateFirstLevelConcurrencyException : Exception
```

Comment:
```csharp
public class MyException : Exception
    {
        public MyException() { }
        public MyException(string message) : base(message) { }
        public MyException(string message, Exception inner) : base(message, inner) { }
        protected MyException(
          System.Runtime.Serialization.SerializationInfo info,
          System.Runtime.Serialization.StreamingContext context) : base(info, context) { }
    }
```

Reply:
I guess this is a confirmation that everything is OK. Resolving